Resources/Prototypes/_NF/Catalog/Fills/Backpacks/StarterGear/backpack.yml

```diff
@@ -36,6 +36,20 @@
       - id: AccessConfigurator
       - id: PhoneInstrument
 
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackIan
+  id: ClothingBackpackIanSrFilled
+  components:
```

Comment:
Can probably multi parent this off the backpack

Reply:
I guess

Reply:
Done